lib/datadog/tracing/contrib/roda/patcher.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+# typed: ignore
+
+require_relative 'patcher'
+require_relative 'ext'
+require_relative 'instrumentation'
+
+module Datadog
+  module Tracing
+    module Contrib
+      module Roda
+        # Patcher enables patching of 'roda'
+        module Patcher
+          include Contrib::Patcher
+          ::Roda.use Datadog::Tracing::Contrib::Rack::TraceMiddleware if defined? ::Roda
```

Comment:
We should remove this top-level call and instead, due to lack of a good way to automatically instrument Roda+Rack, we should add this manual step to the readme.